src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  class Calculator {
+    constructor() {
+      this.result = 0;
+    }
```

Comment:
just
```suggestion
    result = 0;
```
Because you don't have any logic in the constructor. So you don't need it